contracts/test/PolynomialEval.t.sol

```diff
@@ -22,13 +22,11 @@ contract PolynomialEval_newEvalDomain_Test is Test {
             cmds[2] = vm.toString(logSizes[i]);
 
             bytes memory result = vm.ffi(cmds);
-            (uint256 sizeInv, uint256 groupGen, uint256 groupGenInv) =
-                abi.decode(result, (uint256, uint256, uint256));
```

Comment:
why don't we change `diff-test/main.rs::NewPolyEvalDomain` to remove returning this instead?

Reply:
addressed in [78267a8](https://github.com/EspressoSystems/espresso-sequencer/pull/1840/commits/78267a87ece4066ad218ed4073b1a7e22645600d)